hbase-server/src/test/java/org/apache/hadoop/hbase/client/TestFromClientSide5.java

```diff
@@ -2274,6 +2274,21 @@ public void testCellSizeLimit() throws IOException {
         // expected
       }
     }
+    //test cell size no limit
+    final TableName noLimitTable = TableName.valueOf("testCellSizeNoLimit");
```

Comment:
Use the `@Rule` provided in this class instead of a String for the table name, such as `name.getMethodName()`.

Reply:
Good suggestion, done.